00-RELEASENOTES

```diff
@@ -9,6 +9,36 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 8.0.3  -  Released Wed 23 Apr 2025
+================================================================================
+
+Upgrade urgency SECURITY: This release includes security fixes we recommend you
+apply as soon as possible.
+
+Bug fixes
+=========
+* Optimize RDB load performance and fix cluster mode resizing on replica side (#1199)
+* Fix memory leak in forgotten node ping ext code path (#1574) 
+* Fix cluster info sent stats for message with light header (#1563)
+* Use fake client flag to replace not conn check (#1198)
```

Comment:
This line mentions internal details, not very useful to the users. If it fixes an actual problem (does it?), we can mention that, or if it's just refactoring we don't need to mention it IMHO.

```suggestion
```

Reply:
Sounds good. Removed